os2forms_failed_jobs.services.yml

```diff
@@ -1,4 +1,13 @@
 services:
   Drupal\os2forms_failed_jobs\Helper\Helper:
-    arguments:
-      - '@database'
+    arguments: [
+      '@entity_type.manager',
+      '@database',
+      '@logger.factory'
+    ]
+
+  Drupal\os2forms_failed_jobs\EventSubscriber\AdvancedQueueEventsSubscriber:
+    class: Drupal\os2forms_failed_jobs\EventSubscriber\AdvancedQueueProcessSubscriber
```

Comment:
The `class` property is superfluous when using the full class name as the service name.

Reply:
Removed property